modules/effects/src/effect_notification.ts

```diff
@@ -4,7 +4,7 @@ import { Notification, Observable } from 'rxjs';
 
 export interface EffectNotification {
   effect: Observable<any> | (() => Observable<any>);
-  propertyName: string;
+  propertyName: PropertyKey;
```

Comment:
Should this be `EffectPropertyKey`?

Reply:
It probably should, but that would mean we'd have to re-type the effects_resolver and so on. Let's take it bit by bit.